packages/application-shell/src/components/gtm-booter/gtm-booter.js

```diff
@@ -3,9 +3,9 @@ import PropTypes from 'prop-types';
 import * as gtm from '../../utils/gtm';
 import defaultTrackingEventWhitelist from '../../tracking-whitelist';
 
-const mcngShape = PropTypes.shape({
-  track: PropTypes.func.isRequired,
-  getHierarchy: PropTypes.func.isRequired,
+export const GtmContext = React.createContext({
```

Comment:
This was originally planned to contain "everything" an application might need. Now that we established `ApplicationContext` we don't need this anymore and the renaming is the right thing to do 👍 

Nice to see that we finally ended up where we planned to go over a year a go 🌈